app/views/provider_interface/application_choices/index.html.erb

```diff
@@ -45,7 +45,10 @@
     <% if @application_choices.any? %>
       <div class="app-application-cards">
         <% @application_choices.each_with_index do |choice, index| %>
-          <%= render ProviderInterface::ApplicationCardComponent.new(application_choice: choice, sort_by: @page_state.sort_by) %>
+          <%= render ProviderInterface::ApplicationCardComponent.new(
+            application_choice: choice,
+            show_date: @page_state.sort_by == 'Days left to respond' ? :reject_by_default_at : :updated_at,
```

Comment:
Would it be tidy to map this in `PageState`? Then it's the only place for this mapping to live.